utils_cv/tracking/dataset.py

```diff
@@ -1,39 +1,51 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 
+import os
+import os.path as osp
 from typing import Dict
 from torch.utils.data import DataLoader
 from torchvision.transforms import transforms as T
 from .references.fairmot.datasets.dataset.jde import JointDataset
+from .opts import opts
 from ..common.gpu import db_num_workers
 
 
 class TrackingDataset:
     """A multi-object tracking dataset."""
 
     def __init__(
-        self, data_root: str, train_paths: Dict, batch_size: int = 12
+        self,
+        data_root: str,
+        name: str = "default",
+        batch_size: int = 12,
+        down_ratio: int = 4,
```

Comment:
Should we not down-size by default, ie set the default to 1 here?

Reply:
I'm actually going to remove this as a param in our interface. In the original argparser, the comment is "output stride. Currently only supports 4." 😶